### ISSUE TITLE
Show running version in UI

### ISSUE THREAD

**Issue by hwinkel** [jaegertracing/jaeger-ui#365]
Is there a way to show the actual version (or git hash before release)  (UI, Server) in the UI ?

**Comment by jan25** [comment]
Is this still a valid issue? If no one is working on this, i could pick this up

**Comment by tiffon** [comment]
@jan25 This is still a valid issue, and it would be awesome if you could pick it up!

For now, the version of the UI is available to JS, but it is not displayed anywhere. Maybe as a footer on the sear h page would work? Open to other approaches, too!

The version of jaeger-query would need to be injected, by jaeger-query, before it is available to JS. Similar to the base path. So that's a bit of work that needs to be done in jaeger core.

**Comment by jan25** [comment]
@tiffon Footer would be a good idea. What do you think about putting it under 'About Jaeger' dropdown?
Or i was also thinking like a tiny badge near 'Jaeger UI' title in nav bar, it would show UI version and when you hover over badge the tiny tooltip may show the jaeger-query version.

I found a way to inject jaeger-query version from StaticFileHandler on server side. We are currently using regex replace to inject stuff into index.html from there. Would you know if there are better ways to inject data from Go into html?

**Comment by alanisaac** [comment]
I recently ran into this issue, struggling to figure out what version of the UI I had running without digging into my infrastructure / deployment.  I took a quick stab at implementing the suggestion to put the UI version in the "About Jaeger" dropdown, and opened #606.

I'm more than happy to play around with other ideas, and / or continue on with trying to enhance it with the commit hash or jaeger-query version.
